lib/exception_handling/exception_info.rb

```diff
@@ -82,9 +82,10 @@ def honeybadger_context_data
     end
 
     def controller_name
-      @controller_name ||= if @controller
-                             @controller.request.parameters.with_indifferent_access[:controller]
-                           end.to_s
+      @controller_name ||= (
```

Comment:
`controller_name` isn't actually accurate here anymore, as it could be either the controller name _or_ the honeybadger grouping.

Also, this sets the `controller` param on honeybadger, which we are using to do grouping.

We can accomplish a similar thing if we, say, set this as the [Honeybadger fingerprint](https://docs.honeybadger.io/lib/ruby/getting-started/customizing-error-grouping.html#customizing-the-grouping-for-honeybadgernotify) which should accomplish the same thing without overloading `controller_name`

Thoughts?

Reply:
`fingerprint` will completely overhaul the grouping that already happens by Honeybadger. By utilizing the `controller` keyword param for `Honeybadger#notify`, we can keep using HoneyBadger's `fingerprint`, with a slight modification to it by passing in the `controller` keyword arg.

> The default information we use to group errors is:
> 1. The first line of the stack trace otherwise
> 2. The name of the error
> 3. The component/controller name (if configured)

Reply:
Ah, yeah, I read it wrong then. Was hoping it would sit on top of it.

Sounds good then